src/confd/bin/load

```diff
@@ -56,4 +56,7 @@ fi
 logger -sik -p user.notice "Loaded $fn successfully."
 if [ "$config" = "failure-config" ]; then
     banner_append "ERROR: Corrupt startup-config, system has reverted to default login credentials"
+else
+    # Ensure correct ownership, in particular after factory reset
+    chown admin:wheel "$fn"
```

Comment:
(This might have been an issue before this change as well, but...)

Are there any guarantees that `admin` is a valid username here? Maybe `root:wheel` to reflect that it was created by the system?

Reply:
Ah, good point!  As long as its got `-rw-rw---` any user in the `admin` *group* should be able to work it.  Thanks!

Reply:
Fixed!